visualize

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
```

Comment:
I would love for this functionality to exist as a flag to the Kingpin script... so that a simple `dry` (or not `dry`!) run of Kingpin would automatically generate the output and help the user see it. Imagine if the generated data included representations of how long each step took, allowing for the graph to be used to investigate and optimize various parts of your deployment process.

Furthermore, it could also be used to much more easily pinpoint a failure -- it could output the HTML no matter what (as long as the `--graph` flag was supplied), and the actors that failed permanently could show up in RED, actors that failed (but `warn_on_failure=true`) could show up in YELLOW and all other actors could show up in blue or green.

That would be super useful!